shared/actions/fs/index.js

```diff
@@ -320,6 +323,71 @@ const loadMimeTypeResult = (mimeType: string, action: FsGen.MimeTypeLoadPayload)
     })
   )
 
+const inboxQuery = {
+  ...ChatConstants.makeInboxQuery([]),
+  computeActiveList: false,
+  tlfVisibility: RPCTypes.commonTLFVisibility.any,
+}
+
+const {team, impteamnative, impteamupgrade} = RPCChatTypes.commonConversationMembersType
+
+function* loadResets(action: FsGen.LoadResetsPayload): Saga.SagaGenerator<any, any> {
+  // TODO: maybe uncomment?
+  // const conversations = yield Saga.call(
+  //   RpcChatTypes.localFindConversationsLocalRpcPromise,
+  //   action.payload.tlfName
+  // )
+  const resetRpc = new EngineRpc.EngineRpcCall(
```

Comment:
How expensive is this call? We are calling it every time user enters a TLF list or a TLF; it seems getting the entire inbox may be an overkill. Not for this PR, but perhaps there can be a RPC for service to call, to notify us about TLF resets. WDYT?

Reply:
Yeah I agree, and that's what we do for the team badging. Long term it'd be better to do something like that.